src/base.html

```diff
@@ -165,7 +167,7 @@
 
     <!-- Progressive Web App Manifest -->
     {% if config.extra.manifest %}
-      <link rel="manifest" href="{{ config.extra.manifest | url }}">
+      <link rel="manifest" type="application/manifest+json" href="{{ config.extra.manifest | url }}" crossorigin="use-credentials">
```

Comment:
Could you elaborate on this change?

Reply:
yes so sometimes (for us because of corp auth) the mimetype of the file was not properly set and crossorigin makes sure it works with CORS (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)

Reply:
I would assume that a manifest file would be hosted on the same domain, so is there really need for CORS?

Reply:
For us there was because of a redirect, not sure if it's generally useful or having it "doesn't hurt" either. I can just remove it if you want.

Reply:
I'm just unsure whether this might be a potential security leak. From my knowledge `use-credentials` means: send cookies to the foreign host.

Reply:
That is correct, yeah. Not sure if leak though? I just checked and the above linked spec even mentions the requirement for manifest even if it's on the same domain.

```
The use-credentials value must be used when fetching a manifest that requires credentials, even if the file is from the same origin.
```

Reply:
As I understand it's relevant for when HTTP Basic Auth is required, so I'm ok with this.